### ISSUE TITLE
Blocked app screen for multiple apps

### ISSUE THREAD

**Issue by rachelg98** [ywj7373/MC-group2#93]
Epic #2
Create blocked app screen that can display multiple blocked apps with corresponding unblocking requirements (remaining countdown and step count)

- [x] Dynamic scroll view for list of blocked apps (mutable skeleton)

- [ ] Live updates for changes to recyclerView

- [x] Display block status, block countdown timer, usage time

- [ ] Swipe view for list of blocked apps

**Comment by rachelg98** [comment]
Current workaround
- Refresh app blocking fragment when an app is unblocked to update fragment view
- Disable refresh when hasWindowFocus( ) is false and refresh onResume to prevent service turning off 
- notifyDataChange to adapter to update step counts when change detected 

Swipe view was more of a nice-to-have feature and with the limitations in time, decided not to implement it.
